chart/values.yaml

```diff
@@ -11,3 +11,12 @@ coreDnsOverrides: |
 # Enable the following to use a custom gateway
 # customGateway:
 #   domainName: uds2.dev
+
+passthrough:
+  enabled: false
+  upstreams: []
+    # Add any passthrough upstreams here
+    # items are the subdomain names, they should be unique
+    # e.g.,
+    # - mysubdomin
+    # - myothersubdomain
```

Comment:
nit again but might lean towards calling these subdomains/hosts instead of upstreams? If you make this change just make sure the templates, values, and docs all reflect as well.